packages/bogue-tutorials/bogue-tutorials.0.2.0/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bogue Tutorials"
+description: "Several tutorials for learning how to create GUIs with Bogue"
+maintainer: ["San Vũ Ngọc"]
+authors: ["San Vũ Ngọc"]
+license: ["CC-BY-NC-4.0" "Apache-2.0"]
+tags: ["gui" "bogue" "tutorial"]
+homepage: "https://github.com/sanette/bogue-tutorials"
+doc: "https://sanette.github.io/bogue-tutorials/bogue-tutorials/index.html"
+bug-reports: "https://github.com/sanette/bogue-tutorials/issues"
+depends: [
+  "dune" {>= "3.2" & build}
```

Comment:
```suggestion
  "dune" {>= "3.2"}
```

Reply:
As per the lint: `The package tagged dune as a build dependency. Due to a bug in dune (https://github.com/ocaml/dune/issues/2147) this should never be the case. Please remove the {build} tag from its filter.`